easybuild/easyconfigs/u/UCX/UCX-1.8.0-GCCcore-9.3.0-CUDA-11.0.2.eb

```diff
@@ -0,0 +1,62 @@
+# Note:
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'UCX'
+version = '1.8.0'
```

Comment:
Please update to 1.9.0 (released just days ago)?

Reply:
I'd rather not do that for consistency with the rest of the 2020a components.

Reply:
Actually, we should stick to the same version we use as a dep for OpenMPI in `gompi/2020a`, so leave this as is.